src/DssSpell.sol

```diff
@@ -40,154 +44,65 @@ contract DssSpellAction is DssAction {
-        // Increase the ETH-C Stability Fee (SF) by 1.49%, from 5.00% to 6.49%.
-        DssExecLib.setIlkStabilityFee("ETH-C", SIX_PT_FOUR_NINE_PCT_RATE, true);
-
-        // Increase the WSTETH-A Stability Fee (SF) by 1.91%, from 5.25% to 7.16%.
-        DssExecLib.setIlkStabilityFee("WSTETH-A", SEVEN_PT_ONE_SIX_PCT_RATE, true);
-
-        // Increase the WSTETH-B Stability Fee (SF) by 1.91%, from 5.00% to 6.91%.
-        DssExecLib.setIlkStabilityFee("WSTETH-B", SIX_PT_NINE_ONE_PCT_RATE, true);
-
-        // Increase the WBTC-A Stability Fee (SF) by 0.91%, from 5.79% to 6.70%.
-        DssExecLib.setIlkStabilityFee("WBTC-A", SIX_PT_SEVEN_PCT_RATE, true);
-
-        // Increase the WBTC-B Stability Fee (SF) by 0.91%, from 6.29% to 7.20%.
-        DssExecLib.setIlkStabilityFee("WBTC-B", SEVEN_PT_TWO_PCT_RATE, true);
-
-        // Increase the WBTC-C Stability Fee (SF) by 0.91%, from 5.54% to 6.45%.
-        DssExecLib.setIlkStabilityFee("WBTC-C", SIX_PT_FOUR_FIVE_PCT_RATE, true);
-
-        // ---------- Reduce PSM-PAX-A Debt Ceiling & Disable DC-IAM ----------
-        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-8/23445
-
-        // Note: record currently set debt ceiling for PSM-PAX-A
-        (,,,uint256 lineReduction,) = vat.ilks("PSM-PAX-A");
-
-        // Remove PSM-PAX-A from Autoline.
-        DssExecLib.removeIlkFromAutoLine("PSM-PAX-A");
-
-        // Set PSM-PAX-A debt ceiling to 0
-        DssExecLib.setIlkDebtCeiling("PSM-PAX-A", 0);
-
-        // Reduce Global Debt Ceiling? Yes
-        vat.file("Line", vat.Line() - lineReduction);
-
-        // ---------- RETH-A Offboarding Parameters Finalization ----------
-        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-8/23445
-
-        // Set chop to 0%.
-        DssExecLib.setIlkLiquidationPenalty("RETH-A", 0);
-
-        // Set tip to 0%
-        DssExecLib.setKeeperIncentiveFlatRate("RETH-A", 0);
-
-        // Set chip to 0%
-        DssExecLib.setKeeperIncentivePercent("RETH-A", 0);
-
-        // Set Liquidation Ratio to 10,000%.
-        // Note: We are using low level methods because DssExecLib only allows setting `mat < 1000%`: https://github.com/makerdao/dss-exec-lib/blob/69b658f35d8618272cd139dfc18c5713caf6b96b/src/DssExecLib.sol#L717
-        DssExecLib.setValue(MCD_SPOT, "RETH-A", "mat", 100 * RAY);
-
-        // Note: Update collateral price to propagate the changes
-        DssExecLib.updateCollateralPrice("RETH-A");
-
-        // ---------- SBE parameter changes ----------
-        // Forum: https://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-parameter-reconfiguration-update-4/23441
-
-        // Increase bump by 20,000, from 30,000 to 50,000 DAI
-        DssExecLib.setValue(MCD_VOW, "bump", 50 * THOUSAND * RAD);
+        // ---------- Auction Parameter Updates ----------
+        // Forum: https://forum.makerdao.com/t/stability-scope-auction-parameters-changes-1-liquidation-throughput-limit/23508
 
```

Comment:
This section is missing vote url: https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit#gid=770650493&range=H28

Reply:
Yes, including it.